src/main/java/net/phptravels/gui/pages/HomePage.java

```diff
@@ -1,4 +1,4 @@
-package net.phptravels.gui.login;
+package net.phptravels.gui.pages;
 
```

Comment:
In HomePage constructor missing URL
`setPageAbsoluteURL(R.CONFIG.get(Configuration.Parameter.URL.getKey()));`
check carina-demo homepage as an example